src/test/java/net/openhft/chronicle/testframework/internal/VanillaExceptionTrackerTest.java

```diff
@@ -0,0 +1,132 @@
+package net.openhft.chronicle.testframework.internal;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class VanillaExceptionTrackerTest {
+
+    private AtomicInteger resetCounter = new AtomicInteger(0);
+    private Map<ExceptionHolder, Integer> exceptionCounts;
+    private VanillaExceptionTracker<ExceptionHolder> vet;
+
+    @BeforeEach
+    void setUp() {
+        resetCounter = new AtomicInteger(0);
+        exceptionCounts = new HashMap<>();
+        vet = new VanillaExceptionTracker<>(
```

Comment:
It would be better to provide a static constructor method in `ExceptionTracker` and then only expose the interface in the public API. Maybe that is your strategy once we will move them?